modeltestSDK/api.py

```diff
@@ -691,8 +655,8 @@ def get(self, filter_by: list = None, sort_by: list = None) -> Sensors:
         if sort_by is None:
             sort_by = list()
         params = create_query_parameters(filter_expressions=filter_by, sorting_expressions=sort_by)
-        data = self.client.get(self._resource_path, parameters=params)
-        return Sensors.parse_obj([dict(**item, client=self.client) for item in data])
+        data = self.client.get(self._resource_path, parameters=dict(**params, skip=skip, limit=limit))
+        return Sensors([parse_obj_as(Sensor, dict(**_, client=self.client)) for _ in data])
```

Comment:
Use of "_" is normally limited to parameters not used. But does not cause any errors

Reply:
Replaced with "i"